examples/ui.apps/src/main/content/jcr_root/apps/forms-components-examples/components/form/checkboxgroup/_cq_template.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0"
+  jcr:primaryType="nt:unstructured"
+  jcr:title="Check Box Group"
+  fieldType="checkbox-group"
+  type="string[]"
```

Comment:
Can we keep the type as number[] by default

Reply:
Sure